### DISCUSSION TITLE
Info about RT.Start and RT.Stop

### DISCUSSION THREAD

**Discussion by frederikhvh** [vdemichev/DiaNN#959] [General]
Hi Vadim!
I'm in the process of QC'ing LC methods and looking for ways to assess peak shape fronting and tailing.
I found the RT.Start and RT.Stop metrics in the report.tsv output but couldn't find anything specific in the documentation about what they represent. Are these timepoints measured at 10% of max intensity, 50%? Or how so? 

Best regards,
Frederik

**Comment by vdemichev** [comment]
Hi Frederik,

This is indeed undocumented. These are a proxy for peak width (please use .parquet report of 1.8.2 beta 27 or later), but there's no direct formula which translates them to a peak width. Reason is, peaks are often very noisy, and it's unclear how this noise biases the estimation of start/end RT points. Often you might see that the RT stop minus RT start distribution is different for abundant and less abundant peptides - which usually will be purely an artifact due to noise and/or interferences in the mass spec.

Best,
Vadim
